asciidoc/modules/con_mg_changes-in-vertx-redis-client.adoc

```diff
@@ -0,0 +1,48 @@
+[id="updates-in-vertx-redis-client_{context}"]
+= Updates in {VertX} Redis client
+
+This section describes changes in Redis client.
+
+== Removed deprecated term "slave" from Redis roles and node options
```

Comment:
@vietj Can we say the "slave" term is deprecated from roles and node options?

Reply:
yes we can